consensus-types/blocks/execution.go

```diff
@@ -1277,11 +1447,415 @@ func (e executionPayloadDeneb) ValueInGwei() (uint64, error) {
 	return e.gweiValue, nil
 }
 
+// DepositReceipts --
+func (e executionPayloadDeneb) DepositReceipts() ([]*enginev1.DepositReceipt, error) {
+	return nil, consensus_types.ErrUnsupportedField
+}
+
+// WithdrawalRequests --
+func (e executionPayloadDeneb) WithdrawalRequests() ([]*enginev1.ExecutionLayerWithdrawalRequest, error) {
+	return nil, consensus_types.ErrUnsupportedField
+}
+
 // IsBlinded returns true if the underlying data is blinded.
 func (e executionPayloadDeneb) IsBlinded() bool {
 	return false
 }
 
+// executionPayloadHeaderElectra is a convenience wrapper around a blinded beacon block body's execution header data structure.
+// This wrapper allows us to conform to a common interface so that beacon
+// blocks for future forks can also be applied across Prysm without issues.
+type executionPayloadHeaderElectra struct {
```

Comment:
I'd also consider adding unit tests for all the new methods. Here is an example what I did for ePBS. Feel free to copy:

https://github.com/prysmaticlabs/prysm/pull/13914/files#diff-a4ffcfb9e9d5ea43d14f3217b54c9d2502a8b33d05934d07628b07ff79394887R488